x11-misc/xbindkeys/xbindkeys-1.8.6-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Tool for launching commands on keystrokes"
+SRC_URI="http://www.nongnu.org/${PN}/${P}.tar.gz"
+HOMEPAGE="http://www.nongnu.org/xbindkeys/xbindkeys.html"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~arm ppc ppc64 sparc x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris"
```

Comment:
You have added a new keyword `~x86-fbsd`. Have you tested it?  Besides, we have removed support for FreeBSD variants.

Reply:
Oh, no, I think, it just slipped somehow, when I did a copy. Thank you for noticing! I tried to preserve the content of the original ebuild as much, as possible, so that's just an overseen one.